### ISSUE TITLE
why pretrained model layers is different with define in code

### ISSUE THREAD

**Issue by azuryl** [NVIDIA/retinanet-examples#130]
in retinanet-examples/retinanet/backbones/fpn.py lin2 23

self.pyramid6 = nn.Conv2d(channels[2], 256, 3, stride=2, padding=1)
the define is 256 layers

but in pretrained model https://github.com/NVIDIA/retinanet-examples/releases/tag/19.04
retinanet_rn50fpn.zip 
the layers number is 2048
